app/models/agent.rb

```diff
@@ -0,0 +1,7 @@
+class Agent < ActiveRecord::Base
+  has_many :payload_requests
```

Comment:
hi @bfpepper this is looking great but we are both curious about what exactly payload_requests is.  It seemed to us that payloads would be the join table. Are you saying here that :payload_requests would be another table linking the payloads and the payload requests?